data/json/professions.json

```diff
@@ -3895,5 +3895,15 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "local_drug_dealer",
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you. To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" } ],
+    "items": { "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "crack", "crack", "crack", "switchblade" ], "male": [ "briefs" ], "female": [ "sports_bra", "boy_shorts" ] },
```

Comment:
You can put the crack all in one bag, see #47254.

Reply:
thanks for passing that to me, I'll update it (damn, 2 drug prs in an hour)